Animations/Fade/Up.js

```diff
@@ -26,5 +28,5 @@ var FadeUp = {
     'transform': 'translateY(20px)'
   }
 };
-//Use makeTransition function from main Tux Animation module to create wrapping animation componenet and pass in the default params
-module.exports = Animation.makeTransition('FadeUp', FadeUp);
+//Use makeAnimation function from main Tux Animation module to create wrapping animation componenet and pass in the default params
+module.exports = makeAnimation('FadeUp', FadeUp);
```

Comment:
This does not need to have `'FadeUp'` passed into it since we have defined a `className` property in `FadeUp`.